lib/opensusebasetest.pm

```diff
@@ -1292,7 +1292,7 @@ use root-ssh console directly.
 =cut
 
 sub select_serial_terminal {
-    my ($self, $root) = @_;
+    my ($root) = @_;
```

Comment:
If you want to prevent confusion you could use signatures with a mandatory argument but as the parameter "root" is optional this won't easily work. But if you want to make this not a member method I recommend to also move it out of the class so that it's clear that this is a free function, not a member method.

Reply:
Do you think that moving this to `utils` would make a sense?

Reply:
Yes, `utils` would be a good place. But then this PR also needs to refactor all calls to `select_serial_terminal()` throughout the codebase. Preferably in the same commit.

Reply:
Please check the current implementation.